### DISCUSSION TITLE
Caching in github actions

### DISCUSSION THREAD

**Discussion by alephnull** [goreleaser/goreleaser#3789] [Q&A] [answered]
The buildx layer caching is quite adequate for local usage even in DinD scenarios. When hacking on CI, I spend a lot of time waiting on builds that would be a cache hit most of the time.

The [https://github.com/docker/build-push-action](build-push) action does support GHA caching via [https://docs.docker.com/engine/reference/commandline/buildx_build/#cache-to](cache-to).

Since goreleaser supports buildx, is this supported already? This is probably a question for https://github.com/goreleaser/goreleaser-action but discussions are not enabled there.

**Comment by caarlos0** [comment, marked as answer]
AFAIK we don't do anything with remote caching, no... so it'll work just in the context of the same run...

@crazy-max might have more info though
